### DISCUSSION TITLE
[remark] How to do inline code?

### DISCUSSION THREAD

**Discussion by WillowP** [remarkjs/remark#785] [Q&A] [answered]
Hello! I'm getting started with this (amazing) library, and I'm planning on using both inline and block code snippets heavily. However, I cannot seem to figure out how inline code snippets work.

I've recreated the example under `Use custom components (syntax highlight)` in the docs, and it's working properly.

However, there is no example in the docs that shows inline code. If I want to do this:

```js
const mrkdwn = `# My Article
I'm talking about some code that looks like this ~const noOp = () => {}~!`
```

The resulting HTML is just:
```html
<p>I'm talking about some code that looks like this ~const noOp = () =&gt; {}~!</p>
```

I tried looking at the `remark` docs but couldn't seem to find any help there either.

I'd be very grateful if someone could point me in the right direction. Also happy to make a PR to update the docs myself if there's an answer :)

Thanks!

**Comment by wooorm** [comment]
While fenced code blocks can use (three) tildes, inline code can only use ticks: `` `x` ``: https://spec.commonmark.org/0.30/#code-spans

You could switch to using strings with single or double quotes in JavaScript, to use backticks in a string.
Or you must escape them, in a backticked JavaScript string.
Or you could put the markdown in a file

**Comment by ChristianMurphy** [comment, marked as answer]
Thanks for the kind words @WillowP!
Inline code snippets use backticks https://commonmark.org/help/
```js
const mrkdwn = `# My Article
I'm talking about some code that looks like this \`const noOp = () => {}\`!`
```

runnable example: https://codesandbox.io/s/react-markdown-debug-forked-253d7

**Comment by WillowP** [comment]
Wow, thanks for the fast responses. This is working like a dream now.
